flink-streaming-java/src/main/java/org/apache/flink/streaming/api/environment/StreamExecutionEnvironment.java

```diff
@@ -70,7 +70,6 @@
 import org.apache.flink.streaming.api.datastream.DataStreamSource;
 import org.apache.flink.streaming.api.datastream.SingleOutputStreamOperator;
 import org.apache.flink.streaming.api.functions.source.ContinuousFileMonitoringFunction;
-import org.apache.flink.streaming.api.functions.source.ContinuousFileReaderOperator;
```

Comment:
One more thing. Doesn't removal of line brake the java docs for this class (`@Public` so quite important)? 

Replace `{@link ContinuousFileReaderOperator readers}` with full class reference?

Reply:
Yes, but the actual operator type is now hidden in the factory, so I'd rather remove the reference at all and leave just plain "readers".

Reply:
I think a reference to the code might be helpful for some more power users. I would be also fine with referencing `ContinuousFileReaderOperatorFactory` itself, as that wouldn't explode the comment's length.

Reply:
IMO javadoc of public API shouldn't talk about the implementation classes at all, unless there is a need (here there isn't). And "power users" should be able to go inside the method and see the implementation:)

Reply:
Ok, in that case please update the java docs.

Reply:
Done.